README.md

```diff
@@ -43,21 +43,28 @@ pak::pak("Gilead-BioStats/qcthat")
 
 # Business Process Overview
 
+The minimal development process for {qcthat} can be summarized as: "Tests that reference GitHub issues are summarized in a Qualification Report."  
+
+To break this down further, the minimum required steps for {qcthat} implementation are summarized below. For each required step, additional (optional) configuration details for each core step are described in the following sections. Throughout this section 🧑‍💼 is used to indicate a manual task, and 💻 is used to indicate automated tasks. 
```

Comment:
Should we add the 🧑‍💼 and 💻 tags in the steps within the subsections? We'd need to rewrite things more to do that, I think; we don't currently indicate the automations.